src/FakeItEasy.Analyzer.Shared/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+
+  <PropertyGroup>
+    <PackageTags>FakeItEasy;analyzer</PackageTags>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)FakeItEasy.Analyzer.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <DebugType>full</DebugType>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <DebugType>pdbonly</DebugType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Shared source files -->
+    <Compile Include="$(MSBuildThisFileDirectory)../*.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)**/*.cs" />
+    <!-- Analyzer installation scripts -->
+    <Content Include="$(MSBuildThisFileDirectory)tools/*.ps1" Pack="true" PackagePath="tools/%(Filename)%(Extension)" />
+  </ItemGroup>
+
+</Project>

```

Comment:
While I was at it, I extracted some common bits to this file.
The file name might be misleading, though, since it's not a "real" Directory.Build.props (in the sense that it's not implicitly imported). Maybe we should rename it to something else, e.g. `FakeItEasy.Analyzers.Shared.proj`.

Reply:
I'd like to try something. Currently it's difficult to add new files to the projects, because there's no place to do it in the solution explorer. Introducing a shared project might make things easier (I'm not sure it works with the .NET Core SDK though, I need to try it)

Reply:
OK, it works with a shared project, but it's not perfect. Shared projects don't have all the nice features of SDK-style projects, so the files are listed explicitly in the `.projitems` file (we can manually add glob patterns, but VS doesn't really support it so new files are added explicitly, which causes them to be included twice in the build). But at least we can now add files to the right place from Visual Studio.

Let me know what you think.